src/site/content/en/blog/floc/index.md

```diff
@@ -257,10 +257,16 @@ greater-than-typical number of people who visit sites in a sensitive category, t
 removed. Negative financial status and mental health are among the sensitive categories covered by 
 this analysis. 
 
-Websites opt out of FLoC by setting the [Permissions-Policy](https://www.w3.org/TR/permissions-policy-1/#introduction) 
-header `interest-cohort=()`. For the [FLoC origin trial](https://developer.chrome.com/origintrials/#/view_trial/213920982300098561) 
-in Chrome 89, websites that don't opt out will be included in the FLoC calculation if Chrome detects 
-that they are sites [which load ads or ads-related resources](https://github.com/WICG/floc/issues/82). ([Ad Tagging in Chromium](https://chromium.googlesource.com/chromium/src/+/master/docs/ad_tagging.md) explains how Chrome's ad detection mechanism works.) Websites can also, of course, simply not access or record the cohort IDs of their visitors.
+Websites can exclude a page from the FLoC calculation by setting a [Permissions-Policy](https://www.w3.org/TR/permissions-policy-1/#introduction) header `interest-cohort=()` for that page. For pages that haven't 
```

Comment:
Is `interest-cohort` part of the FLoC project? I can't find this is implemented or specified.

Reply:
> Is `interest-cohort` part of the FLoC project? I can't find this is implemented or specified.

Yes: [github.com/WICG/floc#opting-out-of-computation](https://github.com/WICG/floc#opting-out-of-computation).

Reply:
Have you the header elsewhere in the article? If you haven't, you should, or you can link to the section in the explainer.

Reply:
I've added a link.